spec/factories/state_file_id_intakes.rb

```diff
@@ -108,7 +108,7 @@
     end
```

Comment:
https://stackoverflow.com/a/41626919
I think we're using `transient` wrong here
>transient attributes allow you to pass in data that isn’t an attribute on the model.

All these traits (except for `filing_status` which makes sense here) should not be in this transient block.